### PR TITLE
Publishing updates for CC 18.3

### PR DESCRIPTION
**Pull request by ctauchen**
Product Version(s):

Issue:

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [ ] Deploy preview inspected wherever changes were made 
- [ ] Build completed successfully
- [ ] Test have passed

### CODE REVIEW

File: calico-cloud_versioned_docs/version-18-2/releases.json
Comment (ctauchen) [inline calico-cloud_versioned_docs/version-18-2/releases.json:1]: @tmjd PTAL

File: calico-cloud_versioned_docs/version-18-2/variables.js
Comment (ctauchen) [inline calico-cloud_versioned_docs/version-18-2/variables.js:1]: @tmjd PTAL

File: calico-cloud_versioned_docs/version-18-2/releases.json
Comment (tmjd) [reply calico-cloud_versioned_docs/version-18-2/releases.json:1]: This looks good to me.
AFAIK the only thing we use from this file in the CC docs is for the honeypod images.

File: calico-cloud_versioned_docs/version-18-2/variables.js
Comment (tmjd) [reply calico-cloud_versioned_docs/version-18-2/variables.js:1]: LGTM too

**Review by tmjd — APPROVED** [review]
LGTM

**Pull request merged**